contents/docs/experiments/common-questions.mdx

```diff
@@ -1,46 +1,44 @@
 ---
-title: Common questions about experiments
+title: Troubleshooting and FAQs
 ---
 
-
 ## How do I use an existing feature flag in an experiment?
 
-You generally don't want this, because experiments expect flags to be in a specific format. And if your flag isn't in the precisely correct format, you'll need to change your code anyway, so it's better to use a fresh flag.
+We generally don't recommend this, since experiment feature flags need to be in a specific format (see below) or otherwise they won't work.
 
-However, if you know what you're doing, you can delete your existing flag and use the same key when creating the experiment. This will work.
+However, if you insist on doing this (for example, you don't want to make code change), you can do this for **multiple variant feature flags only** by doing the following:
 
-### But I've already created the flag to roll the feature out and just want to run an experiment on it
+1. Delete the existing feature flag you'd like to use in the experiment from the [PostHog dashboard](https://app.posthog.com/feature_flags)
+2. Create a new experiment and give your feature flag the same key as the feature flag you deleted in step 1.
+3. Name the first variant in your new feature flag 'control'.
 
-Deleting the flag is equivalent to it being disabled, so it will be off for however long it takes you to create the draft experiment. The flag is enabled as soon as the experiment is created (not launched), so this shouldn't disrupt your process too much.
+![Reuse an existing feature flag for an experiment](../../images/features/experiments/reuse-feature-flag-for-experiment.png)
 
-### How do I run a second experiment using the same feature flag as the first experiment? I don't want to delete experiment #1.
+> **Note:** Deleting a flag is equivalent to disabling it, so it is off for however long it takes you to create the draft experiment. The flag is enabled as soon as you create the experiment (not launched).
 
-The same caveats as above apply. If you want to re-run an experiment (using the same feature flag key) while preserving the previous experiment results, simply delete the existing feature flag (not the experiment) and use the same key in the new experiment.
+## How do I run a second experiment using the same feature flag as the first experiment?
 
----
+This is similar to running an experiment using an [existing feature flag](/docs/experiments/common-questions#how-do-i-use-an-existing-feature-flag-in-an-experiment). If you want to re-run an experiment (using the same feature flag key) while preserving the previous experiment results, delete the existing feature flag (not the experiment) and use the same key in the new experiment.
```

Comment:
```suggestion
This is similar to running an experiment using an [existing feature flag](#how-do-i-use-an-existing-feature-flag-in-an-experiment). If you want to re-run an experiment (using the same feature flag key) while preserving the previous experiment results, delete the existing feature flag (not the experiment) and use the same key in the new experiment.
```
Since it is the same doc, I think you can do this.